src/app/child-dev-project/attendance/attendance-calendar/attendance-calendar.component.ts

```diff
@@ -143,6 +165,16 @@ export class AttendanceCalendarComponent implements OnChanges {
     await this.entityMapper.save(this.selectedEvent);
   }
 
+  newNote(): EventNote {
+    const note = new EventNote();
+    note.date = this.selectedDate.toDate();
+    note.children = this.activity.participants;
+    note.authors = this.activity.assignedTo;
+    note.category = this.activity.type;
+    note.subject = this.activity.title;
+    return note;
+  }
```

Comment:
The newly created events are not saved correctly.

To reproduce:

1. Select empty date
2. Create event
3. Add some attendance
4. Click `save`
5. Leave page and open activity again
6. No attendance is visible for the previously selected date